angrmanagement/plugins/log_human_activities/log_human_activities.py

```diff
@@ -0,0 +1,168 @@
+import logging
+import threading
+from ..base_plugin import BasePlugin
+from angrmanagement.config import Conf
+import angrmanagement.ui.views as Views
+from time import sleep
+from getmac import get_mac_address as gma
+
+l = logging.getLogger(__name__)
+l.setLevel('INFO')
+
+try:
+    from slacrs import Slacrs
+    from slacrs.model import HumanActivity, HumanActivityEnum
+except ImportError as ex:
+    Slacrs = None  # type: Optional[type]
+    HumanActivityVariableRename = None  # type: Optional[type]
+    HumanActivityFunctionRename = None  # type: Optional[type]
+
+
+class LogHumanActivitiesPlugin(BasePlugin):
+    def __init__(self, *args, **kwargs):
+        if not Slacrs:
+            raise Exception("Skipping LogHumanActivities Plugin. Please install Slacrs to Initialize it.")
+        super().__init__(*args, **kwargs)
+        self.session = None
+        self.project_name = None
+        self.project_md5 = None
+        self._log_list = list()
+        self.user = gma()
+
+    def on_workspace_initialized(self, workspace):
+        self.slacrs_thread = threading.Thread(target=self._commit_logs, args=(self))
```

Comment:
`args=` is useless here. You will probably see a warning message since `self._commit_logs()` does not take extra arguments.